ios/FluentUI/Card/CardView.swift

```diff
@@ -21,22 +21,138 @@ public enum CardColorStyle: Int, CaseIterable {
     case custom
 }
 
-/// Card style can be horizontal, or vertical
-/// Both styles have a title, an icon and an optional subtitle. The whole card is clickable
-@objc(MSFCardStyle)
-public enum CardStyle: Int, CaseIterable {
-    /// Icon is positioned next to the title and subtitle is positioned below the title
-    case horizontal
-    /// Icon is at the top, title below the icon and subtitle below the title
-    case vertical
+/// Pre-defined sizes of the Card.
+@objc(MSFCardSize)
+public enum CardSize: Int, CaseIterable {
+    case small
+    case large
 
     var width: CGFloat {
         switch self {
-        case .horizontal:
-            return 156
-        case .vertical:
-            return 120
+        case .small:
+            return 164
+        case .large:
+            return 214
+        }
+    }
+    
```

Comment:
Validation check failed here:
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI/Card/CardView.swift:38:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)